### PR TITLE
login-created-and-css-added

### PR DESCRIPTION
**Pull request by Htchienkoua**
Added a background-image and login form... will soon add validators to the form

### CODE REVIEW

File: src/app/login/login.component.html
[the remark this answers is not in the document]
Comment (Htchienkoua) [reply src/app/login/login.component.html:1]: Well... from the form on the backend(which will be the login details from the frontend) , an id will be needed but only in the backend(that is autogenerated)... so i will remove the id entry

[the remark this answers is not in the document]
Comment (kyskiz) [reply src/app/login/login.component.html:1]: id should be generated in backend, not visible in user forms
for login email/password combinatiomn should be enough

also should keep same material input theme

File: src/app/login/login.component.ts [hunk 1]
```diff
@@ -6,5 +6,5 @@ import { Component } from '@angular/core';
   styleUrls: ['./login.component.scss']
 })
 export class LoginComponent {
-
+  value = 'Please insert your Username';
```
Comment (kyskiz) [inline src/app/login/login.component.ts:9]: why is this here? where it is used?

File: src/app/login/mortgage-background-image.jpg
Comment (kyskiz) [inline src/app/login/mortgage-background-image.jpg:1]: image should be put under assets/img directory, reused when possible. 
do not add image for every component

File: src/app/monthly-calc/monthly-calc.component.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<div class="container">
+<body class="container">
```
Comment (kyskiz) [inline src/app/monthly-calc/monthly-calc.component.html:1]: components should not contain body tag

File: src/assets/random.jpg
[the remark this answers is not in the document]
Comment (Htchienkoua) [reply src/assets/random.jpg:1]: used for the not found page

File: src/app/login/login.component.ts (hunk 1 above)
Comment (Htchienkoua) [reply src/app/login/login.component.ts:9]: i am still to implement the tags

**Comment by swoogie** [comment]
pls fix 🙏
Background image is off there's a white gap between the navbar and the image, and the form goes into the navbar on scroll. Login form looks very different to the other forms I suggest copy pasting whatever the forms had and changing the inputs.
